src/client/pages/OfferNew/Checkout/SignDisclaimer.tsx

```diff
@@ -21,54 +23,62 @@ const Wrapper = styled('div')`
     margin: 0;
   }
 `
+
 type Props = {
   offerData: OfferData
 }
 
 export const SignDisclaimer: React.FC<Props> = ({ offerData }) => {
   const textKeys = useTextKeys()
 
+  const currentLocale = useCurrentLocale()
+
+  const temporaryTermsLink = getTermsLink(currentLocale)
+
+  // This link is only temporary since we can't get the correct ones from content-service ATM 👆
+  // Please note that this implementation only works with the cases where we have multiple contracts as long as we don't need to fetch the links from content-service
```

Comment:
Hm, I interpret this solution as it only works when having multiple contracts which might be confusing 🙃

Reply:
Oh, that't the opposite of what I meant to write 🙈